packages/wrangler/src/triggers/deploy.ts

```diff
@@ -340,3 +291,73 @@ export default async function triggersDeploy(
 		logger.log("No deploy targets for", workerName, formatTime(deployMs));
 	}
 }
+
+async function subdomainDeploy(
```

Comment:
Function needs some work to make all tests pass, but should be clearer what's going on with subdomains.